utils/templates.py

```diff
@@ -261,3 +261,55 @@ def __init__(self, params=None, content=None):
 
         self.body = copy.deepcopy(self.params)
         self.body.update({'content': self.content})
+
+class ShortReviewReply(InvitationReply):
+    """
```

Comment:
I agree, but the idea is that the PC can extend the basic configuration and create their own invitations.